pom.xml

```diff
@@ -32,7 +30,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>surveysvc-api</artifactId>
-      <version>10.49.1</version>
+      <version>10.49.3-SNAPSHOT</version>
```

Comment:
Should this be 10.49.2 instead of the SNAPSHOT?

Reply:
I think Richard already did a 10.49.2 release.  For development it should be 10.49.3-SNAPSHOT, then whoever does the release sets it to 10.49.3.   The reason for this is release versions are immutable - if you didn't develop against a SNAPSHOT you'd have to increment the version number every commit.

Reply:
I'd disagree with that. It should be the release version. The api projects create a release artifact on each commit on the assumption they don't change that often

Reply:
I've changed it half a dozen times in the last week